aws_asdi_pipelines/pipelines/amazonia_1/app.py

```diff
@@ -0,0 +1,81 @@
+"""Amazonia1 asdi handler."""
+
+import json
+import os
+import re
+
+import requests
+from aws_lambda_powertools.utilities.data_classes import SQSEvent, event_source
+from stactools.amazonia_1 import create_item
+from stactools.core import use_fsspec
+
+from aws_asdi_pipelines.cognito.utils import get_token
+
+
+def xml_key_from_quicklook_key(key: str) -> str:
+    """
+    Parse quicklook key and return key to INPE's metadata.
+
+    Args:
+        key: quicklook key
+
+    Returns:
+       key to INPE's Amazonia XML
+    """
+
+    match = re.search(
+        r"(?P<satellite>\w+)/(?P<camera>\w+)/"
+        r"(?P<path>\d{3})/(?P<row>\d{3})/(?P<scene_id>\w+)/",
+        key,
+    )
+    assert match, "Could not match " + key
+    qdict = {
+        "satellite": match.group("satellite"),
+        "camera": match.group("camera"),
+        "path": match.group("path"),
+        "row": match.group("row"),
+        "scene_id": match.group("scene_id"),
+        "collection": match.group("satellite") + match.group("camera"),
+    }
+    metadata_key = (
+        "%s/%s/%s/%s/%s/%s_BAND%s.xml"  # pylint: disable=consider-using-f-string
+        % (
+            qdict["satellite"],
+            qdict["camera"],
+            qdict["path"],
+            qdict["row"],
+            qdict["scene_id"],
+            qdict["scene_id"],
+            2,
+        )
+    )
+    return metadata_key
+
+
+@event_source(data_class=SQSEvent)  # pylint: disable=no-value-for-parameter
+def handler(event: SQSEvent,
+            context):  # pylint: disable=unused-argument
+    """Lambda entrypoint."""
+    ingestor_url = os.environ["INGESTOR_URL"]
+    ingestions_endpoint = f"{ingestor_url.strip('/')}/ingestions"
+    headers = {"Authorization": f"bearer {get_token()}"}
+    use_fsspec()
+    for record in event.records:
+        record_body = json.loads(record.body)
```

Comment:
@sharkinsspatial forgot to mention that here I'm assuming a SQS event with possibly more than one message, this differs from what is implemented for the `sentinel1` pipeline. Not sure if both methods are supported. Multiple messages support is specially useful when reconciling a new database with archive data